wordpress-mobile/WordPress-iOS
# Issues/6332 preserve in use content

Fixes #6332
This patch supersedes what was in #6352 for a cleaner model revision.

To test:
Repeat the steps outlined in #6297.
Confirm that the post is not deleted (watch for the NSLog or set a breakpoint), that instead its topic is set to nil, and there is no crash.

Needs review: @koke
